### ISSUE TITLE
Remove the entrypoint from the interface

### ISSUE THREAD

**Issue by grobie** [prometheus/golang-builder#3]
I'd suggest to not set an entrypoint in the Dockerfile but leave that to the caller. The `builder.sh` script should be in the `PATH` so that one can run the docker container with `docker run --rm prom/golang-builder:1.5.3-main builder.sh [options]`. This will also allow to provide more than one command later, for diagnosis for example.

**Comment by sdurrheimer** [comment]
We could yes, but it is already possible to execute another command by using the `--entrypoint` option.

**Comment by grobie** [comment]
Yes, that's what I'm using right now. I've found making the entrypoint always explicit helps the future reader of commands and is also more consistent than to sometimes set the `--entrypoint` and to sometimes rely on a default not immediately apparent.
